code/modules/mining/ores_coins.dm

```diff
@@ -523,4 +523,44 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 /obj/item/coin/adamantine/doubloon
 	name = "doubloon"
 
+/obj/item/coin/eldritch
+	name = "eldritch coin"
+	desc = "Everytime it lands heretic you heal, everytime it lands blade you will get damaged. Favours heretics."
+	icon_state = "coin_heretic"
+	custom_materials = list(/datum/material/diamond = 1000, /datum/material/plasma = 1000)
+	sideslist = list("heretic", "blade")
+	material_flags = NONE
+	/// Amount of damage that will be recieved
+	var/coin_damage = 0
```

Comment:
It looks like there is no reason for this to be a var on the item, as it is reset to 0 every time the proc is called. Just define it in the proc.